.yamllint

```diff
@@ -0,0 +1,4 @@
+extends: default
+
+rules:
+  truthy: disable
```

Comment:
If I'm reading the documentation right we can allow `yes` (and `no`, `true, `false`)